src/components/Navbar/Sidebar.js

```diff
@@ -22,3 +23,22 @@ export const SidebarData = [
     className: "nav-text",
   },
 ];
+
+function Sidebar() {
+  return (
+    <div>
+      {SidebarData.map((item, index) => {
+        return (
+          <li key={index} className={item.className}>
```

Comment:
indexes can be okay to keep in the keys -- just know that sometimes an arrays order can be finicky 👍